src/components/Shell/Shell.js

```diff
@@ -33,7 +38,7 @@ const Shell = ({ title, ...props }) => {
     {
       children: t('shell.button.user.settings.item.themeSwitch'),
       icon: 'customize',
-      onClick: () => alert('activate theme switch dialog'),
+      onClick: () => openThemeSwitch(),
```

Comment:
what if we use directly `themeSwitchRef.current.open()`?

Reply:
good point, I removed it.